lib/puppet/provider/package/chocolatey.rb

```diff
@@ -1,4 +1,5 @@
 require 'puppet/provider/package'
+require File.expandpath(File.join(File.dirname(__FILE__), '..', '..', '..', 'puppet_x', 'chocolatey', 'chocolateyinstall.rb'))
```

Comment:
I tend to prefer to see this as 

~~~ruby
require 'pathname'
require Pathname.new(__FILE__).dirname + '../../' + 'puppet_x/chocolatey/chocolatey_install'
~~~

Drop the .rb :)